codejail/jail_code.py

```diff
@@ -376,3 +379,21 @@ def create_rlimits(effective_limits):
     rlimits.append((resource.RLIMIT_FSIZE, (fsize, fsize)))
 
     return rlimits
+
+
+def save_artifacts(artifacts, save_path):
+    datasets_dest_dir = os.path.join(save_path, 'datasets')
+    os.mkdir(datasets_dest_dir)
+    os.chmod(datasets_dest_dir, 0o777)
+    for artifact_path in artifacts:
+        if artifact_path.endswith('.csv'):
```

Comment:
Why do we only take care of csv files ?
What about json, xlsx, img ?

Reply:
so yeah we can add others but we need to know what folders need to be put into do we even wanna support xlsx , should we support .png and put them inside images folder

Reply:
Oh, I don't think it matters now, let's accept everything so if a need arrive to use image it will work automatically ?